content/docs/concepts/multicluster-configuration/index.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Multicluster Istio
+description: Describes how a service mesh can be configured to include services from more than one cluster.
+weight: 60
+---
+
+Istio is a [service mesh](/docs/concepts/what-is-istio/#what-is-a-service-mesh),
+the fundamental property of which is its ability to monitor and manage
+a network of cooperating microservices under a single administrative domain,
+essentially combining the microservices into a controllable composite application.
+For applications up to a certain size, all of the microservices comprising the
+application can be running on a single orchestration platform (e.g., kubernetes cluster),
+but for many reasons (e.g., scale, redundancy, etc.) most applications will eventually
+need to be distributed and have some of their services running elsewhere.
+
+Istio supports many possible topologies for distributing the services of an
+application beyond a single cluster.
+Services within the mesh can use [service entries](/docs/concepts/traffic-management/#service-entries)
+to access standalone external services or services exposed by another loosely-coupled
+service mesh, a.k.a. *mesh federation*.
+Alternatively, if all the services can be under the same administrative control,
+you can [expand the service mesh](/docs/setup/kubernetes/mesh-expansion/) to include services running
+on VMs or bare metal hosts, or you can combine the services from more than one cluster into a
+single composite service mesh, i.e., a *multicluster mesh*.
+
+## Multicluster service mesh
+
+A multicluster service mesh is defined as a single composite service mesh composed of services
```

Comment:
composite & composed, doesn't sound too good. Maybe replace "composed" with "made up of"?